app/src/main/java/com/github/se/cyrcle/model/user/User.kt

```diff
@@ -11,16 +11,25 @@ package com.github.se.cyrcle.model.user
  * @property profilePictureUrl URL of the profile picture of the user.
  * @property favoriteParkings List of unique identifiers of the user's favorite parkings.
  */
-data class User(
+data class UserPublic(
     val userId: String,
     val username: String,
+    val profilePictureUrl: String = "",
+    // val lastLoginTime: Timestamp? = null,
```

Comment:
I think that `lastLoginTime` should be in `UserDetails`. `numberOfContributionsSpots` and  `userReputationScore` should be public since we want to create a community driven app. Those are both stats that should be displayed to the community since they give a hint about how much credibility a review has based on the user that published it.